qiskit_optimization/runtime/qaoa_client.py

```diff
@@ -72,7 +79,28 @@ def __init__(
                 ansatz, the evaluated mean and the evaluated standard deviation.
             store_intermediate: Whether or not to store intermediate values of the optimization
                 steps. Per default False.
+            use_swap_strategies: A boolean on whether or not to use swap strategies when
+                transpiling. If this is False then the standard transpiler with the given
+                optimization level will run.
+            use_initial_mapping: A boolean flag that, if set to True (the default is False), runs
+                a heuristic algorithm to permute the Paulis in the cost operator to better fit the
+                coupling map and the swap strategy. This is only needed when the optimization
+                problem is sparse and when using swap strategies to transpile.
+            use_pulse_efficient: A boolean on whether or not to use a pulse-efficient transpilation.
+                If this flag is set to False by default. See https://arxiv.org/abs/2105.01063.
+            optimization_level: The transpiler optimization level to run if the swap strategies are
+                not used. This value defaults to 1 in the QAOA runtime.
+            rng_seed: An optional int to set the seed of the random generator that chooses the
+                initial point if no initial point is given.
+
+        Raises:
+            QiskitOptimizationError: if optimization_level is not None and use_swap_strategies
+                is True.
         """
```

Comment:
Could you add one more check to ensure ``reps`` is larger than 0? It might be covered by the runtime but if we use  it to construct the initial point it would be good to check it here.

Reply:
Done here: 1e5ba4f